File: week1/joehyun000/BOJ_1978.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    int count = 0;
    cin >> n;
    for (int i = 0; i < n; i++) {
        int num;
        cin >> num;
        if (num <= 1) {
            continue;
        }
        else if (num == 2) {
            count++;
        }
        else {
            bool isPrime = true;
            for (int j = 2; j * j <= num; j++) {
                if (num % j == 0) {
                    isPrime = false;
                    break;
                }
            }
            if (isPrime) {
                count++;
            }
        }
    }
    cout << count;
    return 0;
}

File: week1/Jhwan02/BOJ1978.py
n = int(input())

count = 0

num = list(map(int, input().split()))

for i in num:
    for j in range(2,i-1):
        if i % j == 0:
            break
    else:
        if i != 1:
            count += 1

print(count)
